src/main/java/com/example/book/store/model/User.java

```diff
@@ -30,4 +38,46 @@ public class User {
     private String lastName;
     @Column(name = "shipping_addresses")
     private String shippingAddress;
+    @ManyToMany(fetch = FetchType.EAGER)
```

Comment:
Using FetchType.EAGER is not the best practice.
Please use FetchType.LAZY and add fetch of roles on a query level